openapi3/loader_test.go

```diff
@@ -96,7 +96,7 @@ func TestResolveSchemaRefWithNullSchemaRef(t *testing.T) {
 	doc, err := loader.LoadFromData(source)
 	require.NoError(t, err)
 	err = doc.Validate(loader.Context)
-	require.EqualError(t, err, `invalid paths: path /foo failed validation for operation POST: found unresolved ref: ""`)
+	require.EqualError(t, err, `invalid paths: path /foo failed validation: validation failed for operation POST: found unresolved ref: ""`)
```

Comment:
Well if you could get the message to be thus that'd be easier to read I think:
> invalid paths: path /foo failed validation: for operation POST: found unresolved ref: ""

Reply:
Sure. I went with:

> invalid paths: path /foo failed validation: operation POST: found unresolved ref: ""

(dropped the `for` from your suggestion)